### PR TITLE
Updated polars version, and defined new exception

### PR DESCRIPTION
**Pull request by SimonG85**

### CODE REVIEW

File: pyo3-polars/Cargo.toml [hunk 1]
```diff
@@ -10,8 +10,8 @@ description = "PyO3 bindings to polars"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-polars = {version = "0.26", default_features=false}
-polars-core = {version = "0.26", default_features=false}
-pyo3 = "0.17"
+polars = { version = "0.27", default_features = false }
+polars-core = { version = "0.27", default_features = false }
+pyo3 = "0.18.1"
 thiserror = "1"
```
Comment (ritchie46) [inline pyo3-polars/Cargo.toml:16]: This must remain arrow2, not arrow

File: pyo3-polars/src/error.rs [hunk 2]
```diff
@@ -24,7 +24,7 @@ impl std::convert::From<PyPolarsErr> for PyErr {
         use PyPolarsErr::*;
         match &err {
             Polars(err) => match err {
-                PolarsError::NotFound(name) => NotFoundError::new_err(name.to_string()),
```
Comment (ritchie46) [inline pyo3-polars/src/error.rs:27]: Why these changes?

Comment (SimonG85) [reply pyo3-polars/src/error.rs:27]: Because NotFound has been deleted:
PolarsError::NotFound(name) => NotFoundError::new_err(name.to_string()),
   |                              ^^^^^^^^ variant or associated item not found in `PolarsError`
and 3 new variants are found: ColumnNotFound, SchemaFieldNotFound, StructFieldNotFound.

File: pyo3-polars/Cargo.toml (hunk 1 above)
Comment (SimonG85) [reply pyo3-polars/Cargo.toml:16]: Sorry, my mistake.

**Comment by caniko** [comment]
Please note this also solves #4

**Comment by SimonG85** [comment]
> Please note this also solves #4
> 
> **Requesting feedback** The issue related to this PR came after #4, yet it got an infinite amount of more attention. Please provide a reason so I can improve my workflow.

If you want to test this pr you can use in your project linking it in your Cargo.toml:
pyo3-polars = { git = "https://github.com/SimonG85/pyo3-polars.git", version = "0.1" }

We are waiting for @ritchie46 feedback!

**Comment by caniko** [comment]
Already doing this.

**Comment by ritchie46** [comment]
Looks good! Thanks for the PR.

**Comment by ritchie46** [comment]
I shall release right away.
